portfolio/src/main/webapp/my-travels/my-travels.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../default.css">
+    <link rel="stylesheet" href="../buttons.css">
+    <link rel="stylesheet" href="my-travels.css">
+  </head>
+  <body>
+    <div class="with-frame" id="travel-photos-box">
+      <img id="matterhorn" src="images/matterhorn.jpg">
+      <img id="zurich" src="images/zurich.jpg">
+      <img id="krakow" src="images/krakow.jpg">
+      <img id="disneyland" src="images/disneyland.jpg">
+      <img id="kolobrzeg" src="images/kolobrzeg.jpg">
+      <img id="berlin" src="images/berlin.jpg">
+    </div>
+    <div id="travel-description-box">
+      <div id="map"> Map placeholder. </div>
+      <div id="comment"> Click a photo to see where it was taken! </div>
+      <a class="button" href="../index.html"> Main menu </a>
+    </div>
+    <script src="travels.js"></script>
+    <script async defer
+    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBGB0pnDqUI6ylxSnxtjDqZlgX0two4pWY&callback=initMap">
```

Comment:
note that this might be a risk if you have your project not correctly configured:
https://stackoverflow.com/questions/33313334/is-it-safe-to-share-on-github-my-google-maps-api-key

I haven't used it so far, but [github secrets](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) might work here, if you want to play around with that.

Reply:
I've restricted the key to be working only on my page.

I thought about github secrets, but since the page is deployed somewhere and HTML code is publicly visible, hiding it on github won't save it from potential abuse.